.mailmap

```diff
@@ -163,12 +210,16 @@ Tim Caswell <tim@creationix.com> <tim@0-26-8-e9-4c-e1.dyn.utdallas.edu>
 Tim Price <timprice@mangoraft.com>
 Tim Smart <timehandgod@gmail.com> <tim@fostle.com>
 Tim Smart <timehandgod@gmail.com> <timehandgod@gmail.com>
+Timothy Leverett <zzzzBov@gmail.com> Timothy <zzzzBov@gmail.com>
 Tom Hughes <tom.hughes@palm.com> <tomtheengineer@gmail.com>
 Tom Hughes-Croucher <tom_croucher@yahoo.com>
+Travis Meisenheimer <travis@indexoutofbounds.com> <tmeisenh@gmail.com>
 Trevor Burnham <trevor@databraid.com> <trevorburnham@gmail.com>
 Tyler Larson <talltyler@gmail.com>
+Viktor Karpov <viktor.s.karpov@gmail.com> vitkarpov <viktor.s.karpov@gmail.com>
 Vincent Voyer <v@fasterize.com>
 Vladimir de Turckheim <vlad2t@hotmail.com>
+vsemozhetbyt <vsemozhetbyt@gmail.com> Vse Mozhet Byt <vsemozhetbyt@gmail.com>
```

Comment:
Sometimes people use different name and/or email address in their commits. Mailmap is a way to keep track of that person's preferred username and email address and all the other usernames and email addresses used. 

https://git-scm.com/docs/git-shortlog#_mapping_authors